ui/app/components/app/asset-list-item/asset-list-item.js

```diff
@@ -93,10 +93,12 @@ const AssetListItem = ({
       className={classnames('asset-list-item', className)}
       data-testid={dataTestId}
       title={
-        <>
-          <h2 className="asset-list-item__token-value">{primary}</h2>
-          <h2 className="asset-list-item__token-symbol">{tokenSymbol}</h2>
-        </>
+        <button className="asset-list-item__token-button" onClick={onClick}>
+          <h2>
```

Comment:
One final suggestion for improving the experience: add a title attribute to this H2 that includes both primary and tokenSymbol so that on hover the full value can be seen.